pkg/provisioner/controller.go

```diff
@@ -124,6 +117,20 @@ func (c *Controller) Start(stopCh <-chan struct{}) error {
 	return nil
 }
 
+func (c *Controller) SetLabels(labels map[string]string) {
```

Comment:
It's a little inconsistent that we have a setter for labels but not for any of the other fields in the Controller struct. I know the reason is to preserve backwards compatibility, but a year from now this may seem odd. I think the only other setter we might want is provisioner name but that's also pretty obvious to pass to `NewProvisioner`... so this is just a rambling comment....

Reply:
I agree but I think it's out of the scope of this PR.  Labels have had significant discussion - I don't want to bog this code down with conversation around custom annotations, finalizers, etc.